vulnerabilities/api.py

```diff
@@ -272,12 +268,18 @@ def bulk_search(self, request):
         purls = request.data.get("purls", []) or []
         purl_only = request.data.get("purl_only", False)
         plain_purl = request.data.get("plain_purl", False)
+        vulnerabilities_only = request.data.get("vulnerabilities_only", False)
```

Comment:
@tdruez what will you like to suggest here? how should we take the request data  input here ?

Reply:
My point is that instead of having 1 URL with many exceptions and variations based on the request data, shouldn't we have different endpoints? I'm not saying that this is the best way, I'm just wondering how we can avoid to much complexity within a single endpoint.